src/test/java/org/opensearch/ad/ratelimit/ColdEntityWorkerTests.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.ad.ratelimit;
+
+public class ColdEntityWorkerTests {
```

Comment:
Why is this empty?

Reply:
Added tests after sending the PR.  Will publish a new PR for that.